src/testPane.ts

```diff
@@ -661,7 +661,7 @@ function getFileToDebug (
     }
   }
 
-  const globOptions = { cwd: enviroPath, absolute: true };
+  const globOptions = { cwd: enviroPath, absolute: true, strict: false };
```

Comment:
I guess setting strict to false is what solves the issue?

Reply:
yep